lib/dbservice_web/controllers/session_controller.ex

```diff
@@ -44,16 +44,25 @@ defmodule DbserviceWeb.SessionController do
   def index(conn, params) do
     query =
       from m in Session,
-        order_by: [asc: m.id],
+        order_by: [desc: m.id],
```

Comment:
Let's make this ordering dynamic. We can add `sort_order` in query param such that if `sort_order` is `asc`, the response will be sorted in ascending order. If `sort_order` is `desc`, result will be sorted in descending order. If `sort_order` in not provided in query, then by default it will be sorted in descending.